doc/conf.py

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+
+import datetime
+import sys
+sys.path.insert(0, '../src/files/')
+
+import pyfingerprint
+
+project = u'PyFingerprint'
+master_doc = 'PyFingerprint'
+author = 'Bastian Raschke <bastian.raschke@posteo.de>'
+copyright = '2014-{}, {}'.format(datetime.date.today().year, author)
+version = pyfingerprint.__version__
```

Comment:
Cool :+1: could you also do it for the `setup.py`? Than we get rid of the "Remember to change also the module version" comment.

Reply:
Yes, is in preparation ;)

Reply:
New PR on new branch or should I use this PR? Also have some outstanding package improvements.